src/test/java/com/bankofspring/hello/HelloWorldTest.java

```diff
@@ -0,0 +1,33 @@
+package com.bankofspring.hello;
+
+import static org.junit.Assert.*;
+
+import javax.naming.Context;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+public class HelloWorldTest {
+
+	private ApplicationContext appContext;
+	
+	@Before
+	public void setup(){
+		appContext = new ClassPathXmlApplicationContext("Spring-Module.xml");
```

Comment:
I don't really understand why yet but I think you are supposed to declare a shutdown hook here also.